src/main/application.ts

```diff
@@ -30,6 +35,8 @@ export class Application {
         this.bindSingleton(Logger, StandardLogger);
         this.bindSingleton(Configuration, EnvConfiguration);
         this.bind(RequestFactory);
+        this.bindAll(Router, [MetricsRouter]);
```

Comment:
What happens If the App doesn’t support http server(i.e. payment-broker-worker)?

Reply:
Nothing 😄 
Basically, these just get registered as bindings, but they don't get instantiated or otherwise impact performance unless injected.
I'm pretty sure static overhead of those is negligible.

Reply:
Metrics cannot be collected in this case - so I guess we need to start http server if we want collect metrics for such applications 😄

Reply:
Correction: metrics aren't _exposed via http_ in such case. Metrics collection and exposure are deliberately decoupled, so you can create `MetricsRegistry` and use it to collect metrics and then expose them in some other way using `registry.report()`.

The same projected usage would fit service-api which can just take `MetricsRegistry` and use it with existing router.